packages/elements-legacy/package.json

```diff
@@ -96,33 +96,33 @@
     "@storybook/theming": "6.5.9",
     "@svgr/rollup": "^6.2.1",
     "@svgr/webpack": "^6.2.1",
-    "@typescript-eslint/eslint-plugin": "^5.30.4",
-    "@typescript-eslint/parser": "^5.30.4",
-    "babel-jest": "^28.1.2",
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^5.30.7",
+    "babel-jest": "^28.1.3",
     "babel-loader": "^8.2.5",
     "concurrently": "^6.5.1",
     "cross-env": "^7.0.3",
     "dashify": "2.0.0",
-    "eslint": "8.19.0",
+    "eslint": "8.20.0",
     "eslint-plugin-prettier": "^4.2.1",
     "fs-extra": "^9.1.0",
-    "jest": "^28.1.2",
+    "jest": "^28.1.3",
     "jest-coverage-badges": "^1.1.2",
     "node-polyfill-webpack-plugin": "^1.1.4",
     "postcss": "^8.4.14",
-    "prettier": "2.2.1",
+    "prettier": "2.7.1",
     "react-is": "^17.0.2",
     "rimraf": "^3.0.2",
-    "rollup": "2.75.7",
+    "rollup": "2.77.0",
     "rollup-plugin-css-only": "^3.1.0",
     "rollup-plugin-scss": "^3.0.0",
     "serverless": "^2.72.3",
     "serverless-s3-deploy": "^0.10.1",
     "serverless-s3-remover": "^0.6.0",
     "serverless-single-page-app-plugin": "^1.0.4",
     "tslib": "^2.4.0",
-    "typescript": "4.3.5",
-    "webpack": "^5.73.0"
+    "typescript": "4.7.4",
```

Comment:
Can't be updated, breaks Storybook, hence tied to an absolute version

Reply:
ma bad, didn't release it was elements-legacy. Doesn't specify in the prompt